src/handlers.js

```diff
@@ -137,14 +147,30 @@ async function statusHandler(message) {
     state = 'failure';
   }
 
-  await build.modify((b) => {
-    b.state = state;
-    b.updated = new Date();
-  });
+  try {
```

Comment:
Same here -- the try/catch ends up making the code hard to read, and I don't think it's especially useful for debugging right now.